deploy/iso/minikube-iso/arch/aarch64/package/cri-dockerd-aarch64/cri-dockerd.mk

```diff
@@ -18,7 +18,10 @@ CRI_DOCKERD_AARCH64_ENV = \
 	GO111MODULE=on \
 	GOPATH="$(CRI_DOCKERD_AARCH64_GOPATH)" \
 	PATH=$(CRI_DOCKERD_AARCH64_GOPATH)/bin:$(BR_PATH) \
-	GOARCH=arm64
+	GOARCH=arm64 \
+	GOPROXY="https://proxy.golang.org,direct" \
+	GOSUMDB='sum.golang.org'\
+	GOOS=linux
```

Comment:
Why the new options are needed?

Reply:
For newer version of go, if we don't set those options, the build will fail.
E.g. after go 1.21 GOPROXY no longer tolerates an empty string when GOMOD111 is turned on 
See https://github.com/golang/go/issues/61928#issuecomment-2157743652